src/styles/previewStyles.less

```diff
@@ -256,6 +256,7 @@
 					color: rgba(255,255,255,0.70);
 					line-height: 20px;
 					outline: none;
+					-webkit-user-select: auto;
```

Comment:
Rather than using the webkit version, does it work if you just use the standard, "user-select" style?